src/components/posts/posts.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { Button, View, Text, Image } from 'react-native';
+import { Header, Icon, ListItem ,Avatar , Card, CheckBox} from 'react-native-elements';
+import * as StyledComponents from '../common/commonStyledElements';
+import i18n from "../../internationalization/i18n"
+import {standartAvatar} from "../../asserts"
+import {
+  NavigationParams,
+  NavigationScreenProp,
+  NavigationState,
+} from 'react-navigation';
+export interface IUserInfo {
+  email: string;
+  username: string;
+}
+
+export interface IProps {
+  navigation: NavigationScreenProp<NavigationState, NavigationParams>;
+  userInfo?: IUserInfo;
+}
+
+export default class Posts extends React.Component<IProps> {
+    public constructor(props: IProps) {
+      super(props);
+    }
+    
+    state =  {
+      postText : "",
+      list : [
+        {
+          email: 'alex',
+          avatar: standartAvatar,
+          postText : "jajajajaNavigationScreenPropNavigationScreenPropNavigationScreenPropNavigationScreenPropNavigationScreenPropNavigationScreenPropNavigationScreenPropNavigationScreenPropNavigationScreenPropNavigationScreenPropNavigationScreenPropNavigationScreenPropNavigationScreenProp",
+          postTime :  new Date(),
+          likeCount : 3,
+          isLiked : false
+        },
+        {
+          email: 'ilya',
+          avatar: standartAvatar,
+          postText : "jajajaja",
+          postTime :  new Date(),
+          likeCount : 3,
+          isLiked : false
+        },
+      ]
+      
+    }
+    handleClick =(i : any) => {
```

Comment:
What is **i**? I think it's index and it should have number type, also bad naming for function.